porch/webhooks/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.18-bullseye as builder
+
+WORKDIR /workspace
+COPY go.mod go.sum ./
+COPY porch/go.mod porch/go.sum porch/
+COPY porch/api/go.mod porch/api/go.sum porch/api/
+
+WORKDIR /workspace/porch/webhooks/
+RUN go mod download
+# Prebuild some libraries to warm the cache
+RUN CGO_ENABLED=0 go build -v \
```

Comment:
I know we do this in the other dockerfile, but do we know whether the improved performance from this is worth the complexity?

Reply:
Since it's not in a separate pod anymore, this is a moot point. I don't know if it's worth the complexity in the controllers Dockerfile. I can't say for sure if it's worth the improved performance but it did strike me as unusual, especially considering that we don't need to build the image that often.